model/fv_mapz.F90

```diff
@@ -869,11 +841,11 @@ subroutine Lagrangian_to_Eulerian(last_step, consv, ps, pe, delp, pkz, pk,   &
 endif        ! end last_step check
 
 ! Note: pt at this stage is T_v
-  if ( remap_t .and. (.not.do_adiabatic_init) .and. do_sat_adj ) then
-! if ( do_sat_adj ) then
+  if ( do_sat_adj ) then
                                            call timing_on('sat_adj2')
 
-#ifdef MAPL_MODE_FIX_SMALL_COND
+!#define MAPL_MODE_FIX_SMALL_COND
+#ifdef MAPL_MODE_FIX_SMALL_COND && USE_COND
```

Comment:
@wmputman I assume you don't want this on by default? You didn't change the `CMakeLists.txt`

Reply:
That's correct for now.